packages/pwa-kit-dev/jest.config.js

```diff
@@ -25,5 +25,19 @@ module.exports = {
         '!scripts/version.js',
         '!src/ssr/server/test_fixtures/**',
         '!src/schemas/**'
-    ]
+    ],
+    moduleNameMapper: {
+        'tsx/cjs/api': '<rootDir>/node_modules/tsx/dist/cjs/api/index.cjs',
+        'src/ssr/server/test_fixtures/node_modules/test-extension/setup-server.js':
+            '<rootDir>/src/ssr/server/test_fixtures/node_modules/test-extension/setup-server.js',
+        'src/ssr/server/test_fixtures/node_modules/ts-extension/setup-server.ts':
+            '<rootDir>/src/ssr/server/test_fixtures/node_modules/ts-extension/setup-server.ts',
+        'src/ssr/server/test_fixtures/node_modules/another-extension/setup-server.js':
+            '<rootDir>/src/ssr/server/test_fixtures/node_modules/another-extension/setup-server.js',
+        'src/ssr/server/test_fixtures/node_modules/extension-with-bad-setup-server/setup-server.js':
+            '<rootDir>/src/ssr/server/test_fixtures/node_modules/extension-with-bad-setup-server/setup-server.js',
+        'src/ssr/server/test_fixtures/node_modules/extension-with-setup-server-no-default-export/setup-server.js':
+            '<rootDir>/src/ssr/server/test_fixtures/node_modules/extension-with-setup-server-no-default-export/setup-server.js'
+    },
```

Comment:
Imagine a world were we have many many extensions, will we have to continually update the dev project like this to make sure things work?